Excluding css files from istanbul

Looking into adding my own css using postcss for rollup. I added the plugin, and got the build to nearly complete, except it fails on one of the last steps concerning istanbul:

> test/**/*.test.js → test/dist/bundle.js...
> [!] (Plugin at pos 4 plugin) SyntaxError: Unexpected token (3:0)
> css/ntk.css (3:0)
> SyntaxError: Unexpected token (3:0)
>     at Parser.pp$5.raise (/Users/shota/javascript/videojs-notes/node_modules/istanbul-lib-instrument/node_modules/babylon/lib/index.js:4454:13)
>     at Parser.pp.unexpected (/Users/shota/javascript/videojs-notes/node_modules/istanbul-lib-instrument/node_modules/babylon/lib/index.js:1761:8)
>     at Parser.pp$3.parseExprAtom (/Users/shota/javascript/videojs-notes/node_modules/istanbul-lib-instrument/node_modules/babylon/lib/index.js:3750:12)
>     at Parser.parseExprAtom (/Users/shota/javascript/videojs-notes/node_modules/istanbul-lib-instrument/node_modules/babylon/lib/index.js:7238:22)
>     at Parser.pp$3.parseExprSubscripts (/Users/shota/javascript/videojs-notes/node_modules/istanbul-lib-instrument/node_modules/babylon/lib/index.js:3494:19)
>     at Parser.pp$3.parseMaybeUnary (/Users/shota/javascript/videojs-notes/node_modules/istanbul-lib-instrument/node_modules/babylon/lib/index.js:3474:19)
>     at Parser.pp$3.parseExprOps (/Users/shota/javascript/videojs-notes/node_modules/istanbul-lib-instrument/node_modules/babylon/lib/index.js:3404:19)
>     at Parser.pp$3.parseMaybeConditional (/Users/shota/javascript/videojs-notes/node_modules/istanbul-lib-instrument/node_modules/babylon/lib/index.js:3381:19)
>     at Parser.pp$3.parseMaybeAssign (/Users/shota/javascript/videojs-notes/node_modules/istanbul-lib-instrument/node_modules/babylon/lib/index.js:3344:19)
>     at Parser.parseMaybeAssign (/Users/shota/javascript/videojs-notes/node_modules/istanbul-lib-instrument/node_modules/babylon/lib/index.js:6474:20)
> 
> npm ERR! code ELIFECYCLE
> npm ERR! errno 1
> npm ERR! videojs-notes@0.0.0 build:js: `rollup -c scripts/rollup.config.js`
> npm ERR! Exit status 1
> npm ERR! 
> npm ERR! Failed at the videojs-notes@0.0.0 build:js script.
> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR!     /Users/shota/.npm/_logs/2018-10-06T06_55_02_803Z-debug.log
> ERROR: "build:js" exited with 1.
> npm ERR! code ELIFECYCLE
> npm ERR! errno 1
> npm ERR! videojs-notes@0.0.0 build: `npm-run-all -p build:*`
> npm ERR! Exit status 1
> npm ERR! 
> npm ERR! Failed at the videojs-notes@0.0.0 build script.
> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR!     /Users/shota/.npm/_logs/2018-10-06T06_55_02_826Z-debug.log

I believe this is because istanbul is picking up my css file when I import it into one of the src modules. Reading the code for video-generate-rollup-config, would it be okay to put the line that excludes files from Istanbul using settings.excludeCoverage after the line that merges settings.excludeCoverage from options.excludeCoverage? I believe this would help my case.

--- a/index.js
+++ b/index.js
@@ -179,6 +179,10 @@ const getSettings = function(options) {
   // get babel settings from the users provided function or use the defaults
   settings.babel = options.babel ? options.babel(defaultBabel()) : defaultBabel();
 
+  if (options.excludeCoverage) {
+    settings.excludeCoverage = options.excludeCoverage(settings.excludeCoverage);
+  }
+
   // primed plugins
   settings.primedPlugins = {
     babel: babel(settings.babel),
@@ -189,10 +193,6 @@ const getSettings = function(options) {
     uglify: terser({output: {comments: 'some'}}),
     istanbul: istanbul({exclude: settings.excludeCoverage})
   };
-
-  if (options.excludeCoverage) {
-    settings.excludeCoverage = options.excludeCoverage(settings.excludeCoverage);
-  }
 
   if (options.primedPlugins) {
     settings.primedPlugins = options.primedPlugins(settings.primedPlugins);

